lib/config/options/index.ts

```diff
@@ -2158,9 +2162,11 @@ const options: RenovateOptions[] = [
   {
     name: 'prTitle',
     description:
-      'Pull Request title template (deprecated). Inherits from `commitMessage` if null.',
+      'Pull Request title template. Inherits from `commitMessage` if null.',
     type: 'string',
     default: null,
+    deprecationMsg:
+      'Direct editing of prTitle is now deprecated. Please edit commitMessage subcomponents instead as they will be passed through to prTitle.',
```

Comment:
```suggestion
      'Direct editing of prTitle is deprecated. Please edit commitMessage subcomponents instead as they will be passed through to prTitle.',
```

Same comment about listing or linking to the docs for the replacement subcomponents here.